bin/integration-test

```diff
@@ -56,8 +56,11 @@ echo testing: python -m pytest "$@"
 python -m pytest "$@"
 
 cd clients/js
-yarn
-yarn test:run
+
+# moved off of yarn to npm to fix issues with jackspeak/cliui/string-width versions #1314
```

Comment:
what is this `jackspeak/cliui/string-width` issue about?

Reply:
if you google it - there is a bunch about it - eg

https://github.com/isaacs/jackspeak/issues/5